src/main/webapp/WEB-INF/tags/instructor/results/teamPanel.tag

```diff
@@ -22,7 +23,15 @@
         <div class="inline panel-heading-text">
             <strong>${teamName}</strong>                        
         </div>
-        <span class="glyphicon ${!shouldCollapsed ? 'glyphicon-chevron-up' : 'glyphicon-chevron-down'} pull-right"></span>
+        <div class="pull-right">
+            <c:if test="${!isDisplayingTeamStatistics}">
+                <a class="btn btn-warning btn-xs" id="collapse-panels-button-team-${teamIndex}" data-toggle="tooltip" title="Collapse or expand all student panels. You can also click on the panel heading to toggle each one individually.">
```

Comment:
Should the variable be named `isDisplayingTeamStatistics`?
I can't really tell why should this happen if `!isDisplayingTeamStatistics`.
Maybe something related to the view instead?